ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/HealthCheckResponse.java

```diff
@@ -22,23 +22,29 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.collect.ImmutableMap;
 import com.google.errorprone.annotations.Immutable;
+import org.apache.kafka.streams.processor.TaskMetadata;
+
 import java.util.Map;
 import java.util.Objects;
+import java.util.Set;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @Immutable
 public final class HealthCheckResponse {
 
   private final boolean isHealthy;
   private final ImmutableMap<String, HealthCheckResponseDetail> details;
+  private final ImmutableMap<String, Set<TaskMetadata>> metadata;
 
   @JsonCreator
   public HealthCheckResponse(
       @JsonProperty("isHealthy") final boolean isHealthy,
-      @JsonProperty("details") final Map<String, HealthCheckResponseDetail> details
+      @JsonProperty("details") final Map<String, HealthCheckResponseDetail> details,
+      @JsonProperty("taskMetadata") final Map<String, Set<TaskMetadata>> metadata
```

Comment:
I thought about adding it to details and making this its own `Check` but it didn't fit the pattern

Reply:
Well, the healthcheck is checking the query uptime, so it fits, right? Btw, why returning all details of the streams instead of just the aggregated values of all tasks in a query?

Reply:
The Checks return a boolean in the detail. But we have the information to make that boolean until we receive it all in the health check. That is also why we are sending all the details

Reply:
I feel doing a pre-aggregation here might help, but is this gonna be per query or per instance?